website/blog/2023-12-04-0.73-debugging-improvements-stable-symlinks.md

```diff
@@ -0,0 +1,178 @@
+- [Stable Symlink Support in Metro](/blog/2023/12/04/0.73-debugging-improvements-stable-symlinks#stable-symlink-support-in-metro)
+- [Kotlin Template on Android](/blog/2023/12/04/0.73-debugging-improvements-stable-symlinks#kotlin-template-on-android)
+- [Android 14 Support](/blog/2023/12/04/0.73-debugging-improvements-stable-symlinks#android-14-support)
+- [Deprecated Debugging Features](/blog/2023/12/04/0.73-debugging-improvements-stable-symlinks#deprecated-debugging-features)
+- [New Architecture Updates](/blog/2023/12/04/0.73-debugging-improvements-stable-symlinks#new-architecture-updates)
+
+### Breaking Changes
+
+- [Babel Package Renames](/blog/2023/12/04/0.73-debugging-improvements-stable-symlinks#babel-package-renames)
+- [React Native CLI Changes](/blog/2023/12/04/0.73-debugging-improvements-stable-symlinks#react-native-cli-changes)
+- [Other Breaking Changes](/blog/2023/12/04/0.73-debugging-improvements-stable-symlinks#other-breaking-changes)
+
+<!--truncate-->
+
+## Highlights
+
+### Debugging Improvements
+
+The React Native and Hermes teams are committed to improving the debugging experience in React Native. In 0.73, we're happy to share some initial progress out of this ongoing investment.
+
+#### Console Log History in Hermes
+
+`console.log()` is an ever popular way for developers to quickly debug their JavaScript code. In previous releases, console logs in React Native apps would not be recorded until a debugger was connected. This made it hard to observe logs that occur early during app load.
+
+In React Native 0.73 we've addressed this issue. Hermes now captures all `console.log()` calls in the background, which will be sent to the Console tab when a debugger is first connected — matching the debugging experience in web browsers.
+
+#### Updated Debugging Docs
+
+We've refreshed the [Debugging](/docs/debugging) section of our docs, which now includes up-to-date information on how to connect all supported debuggers, more info on React DevTools, and refreshed visuals.
+
+![Overview of the Debugging docs as of 0.73](/blog/assets/0.73-debugging-docs.jpg)
+
+#### Experimental New Debugger
+
+The React Native team is working on a new JavaScript debugger experience, intended to replace Flipper, with a Technical Preview available as of React Native 0.73. The new debugger opens immediately, and features a stripped-back Chrome DevTools UI customized for debugging React Native with Hermes.
+
+:::note
+The new debugger is **experimental** and has some [known issues](https://github.com/react-native-community/discussions-and-proposals/discussions/733) we are actively working to solve in a future release of React Native.
+:::
+
+![The new debugger frontend opened to the "Welcome" pane](/docs/assets/debugging-debugger-welcome.jpg)
+
+Learn more about enabling this experience [in the docs](/docs/next/debugging?js-debugger=new-debugger#opening-the-debugger).
```

Comment:
I think we can skip this(?). Only because I think there's reasonable delta on the debugging parts (timeline-wise) that don't need re-emphasising right here — in addition to the talk closely matching content that is provided directly in this post.

It might make sense to link the RN EU talk against the New Architecture section. Will pursue 👍🏻